plugins/base/src/test/kotlin/expect/ExpectTest.kt

```diff
@@ -1,62 +1,16 @@
 package expect
 
-import org.junit.Test
-import org.jetbrains.dokka.testApi.testRunner.AbstractCoreTest
-import java.nio.file.Files
-import java.nio.file.Path
-import java.nio.file.Paths
-import java.util.concurrent.TimeUnit
+import org.junit.jupiter.api.DynamicTest.dynamicTest
+import org.junit.jupiter.api.TestFactory
 
-class ExpectTest : AbstractCoreTest() {
+class ExpectTest : AbstractExpectTest() {
+    private val ignores: List<String> = listOf(
+        "*.svg"
+    )
```

Comment:
Please also ignore `.css` and `.js` files by default and remove them from expected sources